libr/core/cmd.c

```diff
@@ -6866,6 +6881,7 @@ R_API int r_core_cmd_lines(RCore *core, const char *lines) {
 			r_core_task_yield (&core->tasks);
 		} while ((nl = strchr (data, '\n')));
 		r_cons_break_pop ();
+		printf ("\n");
```

Comment:
Uh?

Reply:
Why not just flush after \r?

Reply:
If this isn't present, the next line that is printed will go on the same line as the progress bar. Is there something else I should be using?